perun-openapi/openapi.yml

```
@@ -2682,6 +2682,14 @@ components:
       in: query
       required: true
 
+    forceDelete:
+      name: forceDelete
```

This name has to be exactly the same as the string that is parsed in the RPC API, otherwise, it would not be used.
```suggestion
      name: force
```